sturdycouscous/Printer.py

```diff
@@ -0,0 +1,173 @@
+from Mongo_Printer_Client import DBClient
+from models import TLS_Record
+
+class Printer():
+    def __init__(self):
+        self.client = DBClient()
+        self.outputfile = open('output.txt', "w")
+
+    def write_to_file(self, output):
+        pass
+        self.outputfile.writelines(output)
+        self.outputfile.close()
+
+    def mongo_query(self, fields):
+        output = []
+        for row in self.client.column.find({}, fields):
+            output.append(row)
+        print(output)
+        return output
+
+    def dump_db(self):
+        for row in self.client.column.find({}, {'url', 'title'}):
+            record = tls_record().parse_record(row)
+            write_to_file(record.dump_dict())
+
+    def print_record_count(self):
+        return len(self.mongo_query({'_id'}))
```

Comment:
here, you want to use mongo db stats engine rather than querrying for everyting and counting
db.domain_info.aggregate( [ **{ $collStats: { count: { } } }** ] )

in pyMongo this wouold be:
```suggestion
pipeline =  [
    def print_record_count(self):
     { $collStats: { count: { } } }
  ]
 return self.collection.aggregate(pipeline['count'])
```

Reply:
or even better:
```suggestion
        return self.collection.count_documents()
```